### PR TITLE
Fix some opset 7 issues

### PR DESCRIPTION
**Pull request by benbarsdell**

### CODE REVIEW

File: builtin_op_importers.cpp [hunk 1]
```diff
@@ -723,8 +740,20 @@ DEFINE_BUILTIN_OP_IMPORTER(DepthToSpace) {
 }
 #endif // NV_TENSORRT_MAJOR >= 4
 
+DECLARE_BUILTIN_OP_IMPORTER(Mul);
```
Comment (qiubit) [inline builtin_op_importers.cpp]: Why is this line needed, even though actual definition of Mul is in line 1091?

File: builtin_op_importers.cpp [hunk 2]
```diff
@@ -1592,8 +1617,20 @@ DEFINE_BUILTIN_OP_IMPORTER(Squeeze) {
...
 #endif // NV_TENSORRT_MAJOR >= 4
 
+DECLARE_BUILTIN_OP_IMPORTER(Add);
 DEFINE_BUILTIN_OP_IMPORTER(Sub) {
   ASSERT(inputs.size() == 2, ErrorCode::kINVALID_NODE);
+  if( inputs.at(0).is_tensor() && inputs.at(1).is_weights() ) {
+    auto weights = inputs.at(1).weights();
+    auto neg_weights = ctx->createTempWeights(weights.type, weights.shape);
+    auto status = apply_unary_function(
+        weights, &neg_weights, nvinfer1::UnaryOperation::kNEG);
```
Comment (yinghai) [inline builtin_op_importers.cpp]: This chunk of code is very similar to the one above except for `kNEG`. Can we have a helper function for that?

Comment (benbarsdell) [reply builtin_op_importers.cpp]: This will be cleaned up in a follow-up PR that I have.
